sdcm/tester.py

```diff
@@ -2953,10 +2953,21 @@ def clean_resources(self):
 
         self.destroy_credentials()
 
+    def save_nodes_schema(self):
+        if self.db_cluster is None:
+            self.log.info("No nodes found in the Scylla cluster")
+
+        self.log.info("Save nodes user schema in the files")
+        for node in self.db_cluster.nodes:
+            node.save_cqlsh_output_in_file(cmd="desc schema", log_file="schema.log")
+            node.save_cqlsh_output_in_file(cmd="select JSON * from system_schema.tables",
+                                           log_file="system_schema_tables.log")
+
     def tearDown(self):
         self.teardown_started = True
         with silence(parent=self, name='Sending test end event'):
             InfoEvent(message="TEST_END").publish()
+        self.save_nodes_schema()
```

Comment:
@fruch , @juliayakovlev 

The ignored request to make the schema gathering be `best-effort` approach instead of the `blocking` one as it is here paid off already in the K8S test runs where K8S logs gathering step was not reached because tearDown function failed to execute CQL commands.

See: https://jenkins.scylladb.com/job/scylla-operator/job/operator-master/job/upgrade/job/upgrade-scylla-k8s-eks-test/11/consoleFull

Reply:
@vponomaryov 
I did not ignored your request. I explained why I did it in the tearDown step.
What is a different in K8S, that cql can't be executed on this step?
It means that for K8S we should another solution. It's not needed to try out all node IPs for other backends (aws, gce..)

Reply:
for now  put the `@silence()` decorator ontop of it

Reply:
> @vponomaryov I did not ignored your request. I explained why I did it in the tearDown step. What is a different in K8S, that cql can't be executed on this step? It means that for K8S we should another solution. It's not needed to try out all node IPs for other backends (aws, gce..)

Some temporary bug with the CQL connectivity breakage.
Gathering logs always must be `best-effort`.
It makes no sense to stop gathering other logs if some failed.